qiita_pet/handlers/api_proxy/tests/test_sample_template.py

```diff
@@ -0,0 +1,170 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from unittest import TestCase, main
+from os.path import join
+
+from qiita_core.qiita_settings import qiita_config
+from qiita_pet.handlers.api_proxy.sample_template import (
+    sample_template_summary_get_req, sample_template_post_req,
+    sample_template_put_req, sample_template_delete_req,
+    sample_template_filepaths_get_req)
+
+
+class TestSampleAPI(TestCase):
+    def test_sample_template_summary_get_req(self):
+        obs = sample_template_summary_get_req(1, 'test@foo.bar')
+        exp = {'summary': {
```

Comment:
Here also applies the above comment about returning `{'status': 'success', 'message': None, 'info': {<previous_result>}}`, please fix here and in any other instance.